mozilla/blurts-server
# Allow scanning for exposures without logging in

# References: 
Jira: MNTOR-1309, MNTOR-1381, MNTOR-1382 and MNTOR-1383.
Figma: https://www.figma.com/file/iJFOikscJeplRBOuKOtYEW/Onboarding?node-id=259-23583&t=c89tLRxJ801kMwlG-4



# Description

This adds the ability for users ~~located in the US (still verifying if that's a requirement - it's not, e6ecf8265)~~ to scan for breaches and sign up afterwards.

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/230082150-6ef3dbb5-8f32-4f9b-8da2-40beb2d10e1f.png)

![image](https://user-images.githubusercontent.com/4251/230082210-a0a36b3a-1cf9-4035-9f4e-93add1e547c2.png)

![image](https://user-images.githubusercontent.com/4251/230082366-486398cd-3607-4e1a-a0c3-7c7019c74464.png)

![image](https://user-images.githubusercontent.com/4251/230082289-d90e3d10-e05c-47fd-9d3b-ac1bf4b72505.png)

# How to test

~~Set your browser language to `en-US`, then~~ enter an email address on the homepage. Make sure to test with an unbreached email, an email with <6 breaches, and an email with more than six breaches.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added. (But not submitted for localisation.)
- [x] All acceptance criteria are met. ~~(Except for available globally - this is still unclear.)~~
